src/identity/platform.rs

```diff
@@ -0,0 +1,97 @@
+//! Kernel cmdline parsing - utility functions
+//!
+//! NOTE(lucab): this is not a complete/correct cmdline parser, as it implements
```

Comment:
At some point I guess I'll start bribing @jlebon to turn this into a proper library, but today is not that day :)

Reply:
Totally agreed! I feel like a lot of code will be similar between `Zincati`/`Afterburn` and `fedora-coreos-pinger`. Like you said, today is not the day ;)